### PR TITLE
Change Nonograms task requirements 

### PR DESCRIPTION
**Pull request by helenakrasnova**
#### Title of Pull Request

fix Nonograms task requirements 

#### 🤔 This is a ...

- [ ] 🌟 New task
- [ ] 🌐 New module
- [X] ⚙️ Update to an existing task
- [ ] 🔧 Update to an existing module
- [ ] 🔗 Update or addition of external resources or links
- [ ] 🐛 Fix in a task or related content
- [ ] 🛠 Fix in a module or related content
- [ ] ✏️ Fixed a typo or grammatical error
- [ ] 🔗 Fixed a broken link
- [ ] ❓ Other (specify: **\*\*\*\***\_\_\_\_**\*\*\*\***)

#### Description

- **Brief Overview:**
  
- **Implementation Approach:**
  

#### Additional Information

- **Screenshots/Links:**
  
- [ ] Related Issues:

#### Checklist

- [x] ✅ I have performed a self-review of my own code.
- [x] 📝 I have commented my code, particularly in hard-to-understand areas.
- [x] 🔧 I have made corresponding changes to the documentation (if applicable).
- [x] 🚫 My changes generate no new warnings or errors.

### CODE REVIEW

**Review by valerydluski — APPROVED** [review]

File: tasks/nonograms/README.md [hunk 1]
```diff
@@ -18,26 +18,26 @@ Here you can learn [how to solve the nonogram](https://nonograms-katana.fandom.c
...
 - the default size of the frame is 5x5. The clues are given at the top and left side of the grid. The sequence of numbers must be logically arranged and help the player solve the nonogram
-- every 5 cells in a rows and columns (both player's field and clues) should be divided by divider (bold line).
-- a player is able to fill in a square in the grid, using left mouse-click. On a click event, trigger the functionality of changing the color of the grid to dark (black). When player clicks on dark cell - it will change to empty (white).
-- end game when players fill all **black** squares correctly according to the hints. On a successful game solution, display "Great! You have solved the nonogram in ## seconds!"
+- for player's field: every 5 cells in a rows and columns should be divided by divider (bold line). For clues: every 5 cells in a **rows** should be divided by divider (for left clues); every 5 cells in a **columns** should be divided by divider (for top clues). Clues should be divided from player's field by bold line.
```
Comment (ThorsAngerVaNeT) [inline tasks/nonograms/README.md:21]: I believe it's better to replace `for player's field` with `for game field`. What do you think?

File: tasks/nonograms/README.md [hunk 2]
```diff
@@ -18,26 +18,26 @@ Here you can learn [how to solve the nonogram](https://nonograms-katana.fandom.c
...
+- a player is able to fill in a cell in the grid, using left mouse-click. On a click event, trigger the functionality of changing the color of the grid to dark (black). When player clicks on dark cell - it will change to empty (white).
+- end game when players fill all **black** cells correctly according to the hints. On a successful game solution, display "Great! You have solved the nonogram!" (in case you missed stop-watch implementation)
```
Comment (ThorsAngerVaNeT) [inline tasks/nonograms/README.md:23]: Everywhere `clues` is used, `hints` should be replaced with `clues`, isn't it?

File: tasks/nonograms/README.md [hunk 3]
```diff
@@ -73,18 +73,18 @@ Recommended usage of [eslint (eslint-config-airbnb-base)](https://eslint.org/),
 
 - [ ] layout, design, responsive UI: `+20`
 - [ ] at the beginning state of the game, the frame has size 5x5. The sequence of numbers is logically arranged and help the player solve the nonogram: `+20`
-- [ ] every 5 cells in a rows and columns (both player's field and clues) should be divided by divider `+5`
+- [ ] cells and clues are divided by divides as described in Basic block: `+5`
```
Comment (ThorsAngerVaNeT) [inline tasks/nonograms/README.md:76]: `divides` should be replaced with `divider` if I understand correctly

File: tasks/nonograms/README.md (hunk 2 above)
Comment (helenakrasnova) [reply tasks/nonograms/README.md:23]: I think clues is OK, eventually it is synonims 
```
Hint: A hint is a subtle suggestion or clue that is meant to guide someone towards a particular conclusion or course of action. Hints are often more indirect than tips or advice and may require the recipient to do some interpretation or detective work to understand their meaning. For example, a detective might drop a hint about a suspect's whereabouts to lead their team in the right direction.

Clue: A clue is a piece of evidence or information that helps to solve a mystery or puzzle. Clues are usually more concrete than hints and are often used in the context of a crime or investigation. For example, a detective might find a clue at a crime scene that leads them to identify a suspect.
```

File: tasks/nonograms/README.md (hunk 3 above)
Comment (helenakrasnova) [reply tasks/nonograms/README.md:76]: dividers, you're right

File: tasks/nonograms/README.md (hunk 2 above)
Comment (helenakrasnova) [reply tasks/nonograms/README.md:23]: Make sense, will do

**Pull request merged**